### ISSUE TITLE
False positives for directive comment with description added in ESLint v7.x.

### ISSUE THREAD

**Issue by ota-meshi** [mysticatea/eslint-plugin-eslint-comments#42]
ESLint v7.x supports `/* eslint-disable -- description */` comments, but some rules in this plugin do not support this comments.

e.g.

```js
/*eslint eslint-comments/disable-enable-pair: error */

/*eslint-disable no-undef, no-unused-vars -- description */
var foo = bar()
```

![image](https://user-images.githubusercontent.com/16508807/81529151-306dc100-9399-11ea-9c67-1e61b0a45113.png)
